### PR TITLE
backend-common: bump UrlReader integration test timeout

### PR DESCRIPTION
**Pull request by freben**
Flakiness!

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4354-lwa3xddzf7ztu04uwapbqknsx9hsfwba.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6019470298552d4ff6545097

File: .changeset/brave-crews-taste.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': patch
+---
+
+Bump UrlReader integration test timeout, since they were a bit flaky.
```
Comment (benjdlambert) [inline .changeset/brave-crews-taste.md:5]: Don't think this needs a changeset right?

Comment (benjdlambert) [reply .changeset/brave-crews-taste.md:5]: It's just tests, doesn't change the output of the built artefact?

File: packages/backend-common/src/reading/integration.test.ts [hunk 2]
```diff
@@ -69,6 +69,8 @@ function withRetries(count: number, fn: () => Promise<void>) {
 }
 
 describe('UrlReaders', () => {
+  jest.setTimeout(30_000);
```
Comment (OrkoHunter) [inline packages/backend-common/src/reading/integration.test.ts:72]: Wow, TIL that I could write `30000` like this.

File: .changeset/brave-crews-taste.md (hunk 1 above)
Comment (freben) [reply .changeset/brave-crews-taste.md:5]: Ah yeah, just habits

Comment (OrkoHunter) [inline .changeset/brave-crews-taste.md:5]: In my understanding, we don't really have to add changesets for test updates. But it's fine anyway.

**Review by OrkoHunter — APPROVED** [review]

**Comment by freben** [comment]
updated without changeset

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4354-lwa3xddzf7ztu04uwapbqknsx9hsfwba.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6019470298552d4ff6545097

**Review by Rugvip — APPROVED** [review]

**Pull request merged**